auth/credentials/idtoken/file.go

```diff
@@ -50,38 +52,23 @@ func credsFromBytes(b []byte, opts *Options) (*auth.Credentials, error) {
 		if err != nil {
 			return nil, err
 		}
-		opts2LO := &auth.Options2LO{
-			Email:        f.ClientEmail,
-			PrivateKey:   []byte(f.PrivateKey),
-			PrivateKeyID: f.PrivateKeyID,
-			TokenURL:     f.TokenURL,
-			UseIDToken:   true,
+		var tp auth.TokenProvider
+		if resolveUniverseDomain(f) != internal.DefaultUniverseDomain {
```

Comment:
nit: I usually try to keep the first case positive, `==`, when possible.

Reply:
Fixed, thanks!